Dockerfile

```diff
@@ -1,5 +1,6 @@
 FROM python:3.9.6
 ENV PYTHONUNBUFFERED=1
+ENV DJANGO_SETTINGS_MODULE=opre_ops.settings.local
```

Comment:
Local settings need to be opted into explicitly, I want settings to default to cloud rather than local as a fail-safe.